### ISSUE TITLE
Features Tour

### ISSUE THREAD

**Issue by marySalvi** [atlascope/atlascope#203]
depends on #205 

Create a tour that showcases all functionality and features.

**Possible Waypoints:**
- [x] **Feature**: Embedded Dataset 
        - **imagery/UI**: anatomy image with Vandy slide embedded
        - **text**: explanation of feature and how to interact with dataset
        - this can be several individual waypoints, taking the user "inside" an embedded dataset, etc.
- [x] **Feature**: Pins
        - **Imagery/UI**: Vandy slide with preloaded pins
        - **text**: walk through of types of pins and how to interact with them 
- [x] **Feature**: Jobs - Nucleus Detection
        - **text**: prompt the traveler to take some action (launch job from the UI, or activate the vis feature for (preloaded) job results)
- [x] **Feature**: Job results
- [x] **Feature**:  Jobs - Nucleus to Gland Detection
- [x] **Feature**: Job results
- [ ] **Feature**: VTK
        - **Imagery/UI**: tooth image? @naglepuff would this be the right dataset to use?
        - **text**: describe how to interact with pin to open viewer and a bit about what to expect in new viewer
- [ ] **Feature**: Paraview 
        - **Imagery/UI**: tooth image?
        - **text**: describe how to interact with pin to open viewer and a bit about what to expect in new viewer
- [ ] **Feature**: Similar nuclei (see #208)

**Comment by marySalvi** [comment]
@annehaley @waxlamp @naglepuff @mcovalt 
Please feel free to add features that I missed and any imagery or text that you think would best showcase a feature.

**Comment by annehaley** [comment]
@marySalvi Could you explain this part of the list? Why the sandwich?

>  Feature: Job results
>  Feature: Jobs - Nucleus to Gland Detection
>  Feature: Job results

**Comment by waxlamp** [comment]
> @marySalvi Could you explain this part of the list? Why the sandwich?
> 
> > Feature: Job results
> > Feature: Jobs - Nucleus to Gland Detection
> > Feature: Job results

Mary can confirm or reject when she's back, but I think the idea was to show job results for each of the jobs. It's not a single sandwich, it's two open-face sandwiches: "jobs - nucleus detection" + "job results" followed by "jobs - nucleus to gland detection" + "job results".

**Comment by naglepuff** [comment]
> - Imagery/UI: tooth image? @naglepuff would this be the right dataset to use?

Yes, the tooth dataset would be the one to use for demoing 3D vis. Attaching it as a pin would be the best approach.

**Comment by naglepuff** [comment]
> Feature: Pins
> - Imagery/UI: Vandy slide with preloaded pins
> - text: walk through of types of pins and how to interact with them

In addition to different types of pins, you could show off some features of the pin sidebar. Particularly the select/deselect all button, and zooming to different parts of the map sorts pins by visibility.

You could also try and find a way to demo the pin transitions as you zoom towards/away from pins on the map.

**Comment by marySalvi** [comment]
> > @marySalvi Could you explain this part of the list? Why the sandwich?
> > > Feature: Job results
> > > Feature: Jobs - Nucleus to Gland Detection
> > > Feature: Job results
> 
> Mary can confirm or reject when she's back, but I think the idea was to show job results for each of the jobs. It's not a single sandwich, it's two open-face sandwiches: "jobs - nucleus detection" + "job results" followed by "jobs - nucleus to gland detection" + "job results".

@annehaley @waxlamp  
Yes, I imagined this to be job -> results. I figure we would have info we want to include about the job itself and then different info relating to the results of the job, hence separate waypoints.
